manager/state/raft/raft.go

```diff
@@ -1142,15 +1145,30 @@ func (n *Node) send(ctx context.Context, messages []raftpb.Message) error {
 			continue
 		}
 
+		ch := make(chan struct{})
```

Comment:
I don't think it works to have one channel, because then it's up to the scheduler which write succeeds. With one channel per send, we can specify exactly the order we want for sends.